src/main/kotlin/notary/IrohaCommand.kt

```diff
@@ -63,7 +66,42 @@ sealed class IrohaCommand {
      * @param publicKey public key of signatory
      */
     data class CommandAddSignatory(
-        val accountId: String,
-        val publicKey: String
+            val accountId: String,
+            val publicKey: String
     ) : IrohaCommand()
+
+    /**
+     * Class represents addPeer Iroha command
+     * @param address peer's address, ip and port
+     * @param peerKey peer's key
+     */
+    data class CommandAddPeer(
+            val address: String,
+            val peerKey: ByteArray
+    ) : IrohaCommand() {
+
+        override fun equals(other: Any?): Boolean {
```

Comment:
Do you need equals and hash? Kotlin derivers equals() and hashCode() for data classes.